docs/changelog.md

```diff
@@ -17,7 +17,20 @@ permalink: /changelog/
 * **feature-tabs**
   * Added `TabsUseCases.UndoTabRemovalUseCase` for undoing the removal of tabs.
 * **feature-webcompat-reporter**
-  * Added the ability to automatically add a screenshot as well as more technical details when submitting a WebCompat report.  
+  * Added the ability to automatically add a screenshot as well as more technical details when submitting a WebCompat report.
+* **feature-addons**
+  * `AddonCollectionProvider` now supports configuring a custom collection owner (via user ID or name). ⚠️ This is a breaking change for call sites that don't rely on named arguments.
```

Comment:
nit: Should this be int two separated lines? One for the new functionality and the other for the breaking change api.

Reply:
Hm but it's the same change and we usually keep those together. I agree this look weird though, so I've formatted it differently now in line with what we usually do.

Reply:
👍